src/main/resources/anniv5Resources/localization/eng/dimensiongatepack3/Cardstrings.json

```
@@ -8,28 +8,28 @@
   },
   "${ModID}:Inferno": {
     "NAME": "Inferno",
-    "DESCRIPTION": "Deal !D! damage and apply 10 ${ModID}:Ignite to ALL enemies. NL Gain !M! ${ModID}:Ignite."
+    "DESCRIPTION": "Deal !D! damage and apply !M! ${ModID}:Ignite to ALL enemies. NL Add 2 *Burns to your hand."
   },
   "${ModID}:Minniegun": {
     "NAME": "Minniegun",
     "DESCRIPTION": "Deal !D! damage !M! times. NL Put a *Jam in your discard pile."
```

Minniegun puts the Jam in your draw pile now, right? Looks like the text wasn't updated.
```suggestion
    "DESCRIPTION": "Deal !D! damage !M! times. NL Shuffle a *Jam into your draw pile."
```